core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -164,7 +164,9 @@
             ng-mouseleave="onMouseoutDropdownMenu($event)">
           <li>
             <a href style="padding: 0; width: 200px;" ng-click="onLoginButtonClicked()">
-              <img ng-src="<[getStaticImageUrl('/signin/Red-signin-Long-base-44dp.png')]>" alt="Sign in with Google">
+              <div class="oppia-signin-google">
+                <i class="oppia-signin-google-icon fa fa-google-plus"></i><div class="oppia-signin-google-separator"></div><b>Sign in with Google</b>
```

Comment:
The div tag should move to the next line.
The b tag should move to the line after that.